tests/functional/r/regression/regression_4723.py

```diff
@@ -18,4 +18,4 @@ def play():  # [no-method-argument]
 
 def func():
     with B().get() as b:
-        b.play()
+        b.play()  # [too-many-function-args]
```

Comment:
This was wild. I wrote up in the commit message how I found that changing `as b` to `as boo` was enough to get this message emitted on main, so this result is better. Not exactly sure of the source of the message coupling.

Reply:
wtf !